shell/platform/darwin/ios/framework/Headers/FlutterCodecs.h

```diff
@@ -37,6 +37,7 @@ FLUTTER_EXPORT
                                arguments:(id)arguments;
 @property(readonly, nonatomic) NSString* method;
 @property(readonly, nonatomic) id arguments;
+- (BOOL)isEqualToMethodCall:(FlutterMethodCall*)other;
```

Comment:
The NSObject has an `isEqual:` method. Please override that instead of creating your own. Overriding `isEqual:` and `hash` should enable your participation in standard containers.